src/modules/ai-classification/ai-classification.service.ts

```diff
@@ -59,17 +58,33 @@ export class AiClassificationService {
 
         post =
           await this.postRepository.findPostByIdForAIClassification(postId);
-        const classification =
-          await this.classificationRepository.createClassification(
-            post.url,
-            summarizeUrlContent.response.summary,
-            summarizeUrlContent.response.keywords,
-            folderId,
-          );
 
-        classificationId = classification._id.toString();
+        // 기존에 디폴트 폴더 안막은것들이 있어서... 우선은 다 불러와서 필터링 하는 방식으로 했어용
+        const defaultFolders = await this.folderRepository.getDefaultFolders(
+          post.userId.toString(),
+        );
+        const defaultFolderIds = defaultFolders.map((folder) =>
+          folder._id.toString(),
+        );
+        const isDefaultFolder = defaultFolderIds.includes(
+          post.folderId.toString(),
+        );
         postAiStatus = PostAiStatus.SUCCESS;
+        // 디폴트 폴더인 경우에만 classification 생성
+        if (!isDefaultFolder) {
```

Comment:
```suggestion
        if (isDefaultFolder) {
```
혜온이 코멘트 준것처럼 여기만 바꿔주면 될듯~~ 고생했으